code/crates/core-driver/Cargo.toml

```diff
@@ -26,6 +26,7 @@ malachitebft-core-votekeeper = { workspace = true }
 
 derive-where = { workspace = true }
 thiserror = { workspace = true, default-features = false }
+tracing = { workspace = true }
```

Comment:
```suggestion
tracing = { workspace = true, default-features = false }
```

Reply:
Otherwise this will enable tracing's `std` feature while the driver is currently `no_std`.